backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Views/Home/Index.cshtml

```diff
@@ -1,26 +1,61 @@
 ﻿@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@{
+    bool isProd = string.Equals(Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT"), "prod", StringComparison.OrdinalIgnoreCase);
+}
+
 <!DOCTYPE html>
-<html lang="no" style="background-color: #1EADF7;">
-<head>
-    <base href="~/accessmanagement/ui/" />
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="description" content="Altinn API Delegering">
-    <meta name="keywords" content="Altinn Profil API Delegering">
-    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <link rel="icon" href="https://altinncdn.no/favicon.ico">    
-    <!-- Font Inter support -->
-    <link rel="preconnect" href='https://github.com/Altinn/altinn-cdn'>
-    <link
-      rel="stylesheet"
-      href="https://altinncdn.no/fonts/inter/inter.css"
-    />
-    <title>Altinn</title>
-</head>
-<body>
+
+@if (isProd)
+{
+    <html lang="no" style="background-color: #1EADF7;">
+    <head>
+        <base href="~/accessmanagement/ui/"/>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <meta name="description" content="Altinn API Delegering">
+        <meta name="keywords" content="Altinn Profil API Delegering">
+        <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+        <link rel="icon" href="https://altinncdn.no/favicon.ico">
+        <!-- Font Inter support -->
+        <link rel="preconnect" href='https://rsms.me/'>
+        <link rel="stylesheet" href='https://rsms.me/inter/inter.css'>
+        <title>Altinn</title>
+    </head>
+    <body>
     <div id="root"></div>
     <link rel="stylesheet" type="text/css" href="/accessmanagement/assets/accessmanagement.css" asp-append-version="true"/>
     <script type="module" src="/accessmanagement/assets/accessmanagement.js" asp-append-version="true"></script>
-</body>
-</html>
+    </body>
+    </html>
+}
+else
+{
+    <html lang="no" style="background-color: #1EADF7;">
+    <head>
+        <script type="module">
+        import RefreshRuntime from "http://localhost:5173/@@react-refresh"
+        RefreshRuntime.injectIntoGlobalHook(window)
+        window.$RefreshReg$ = () => {}
+        window.$RefreshSig$ = () => (type) => type
+        window.__vite_plugin_react_preamble_installed__ = true
+        </script>
+        <script type="module" src="http://localhost:5173/@@vite/client"></script>
+        <meta charset="UTF-8"/>
+        <link rel="icon" type="image/svg+xml" href="/src/assets/altinn-logo.svg"/>
+        <!-- Font Inter support -->
```

Comment:
Is it possible to have an env check only on the parts that need to change so we don't get so much duplicated code?